### ISSUE TITLE
A11y: improve a11y capability, adding ARIA roles

### ISSUE THREAD

**Issue by BobH-Official** [konstaui/konsta#155]
### Clear and concise description of the problem

svelte vite plugin returns such warnings:

`[vite-plugin-svelte] /node_modules/.pnpm/konsta@2.0.0/node_modules/konsta/svelte/components/icons/DeleteIcon.svelte:21:2 A11y: <svg> with click handler must have an ARIA role` ... etc.

### Suggested solution

According to MDN: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

It should be easy to add A11y support, by adding a new attribute `role="button"` to the element.

`<div role="button">Title of my button</div>`

### Additional context

Another issue about A11y: https://github.com/konstaui/konsta/issues/92

### Validations

- [X] Follow our [Code of Conduct](https://github.com/konstaui/konsta/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://konstaui.com).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [ ] I'm willing to open a PR
